libs/utils/close.go

```diff
@@ -0,0 +1,13 @@
+package utils
+
+import (
+	"io"
+
+	logging "github.com/ipfs/go-log/v2"
+)
+
+func CloseAndLog(log logging.StandardLogger, name string, closer io.Closer) {
```

Comment:
godoc and mention specifically that it's handy for deferred cases. Might even put defer in the name